src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/ForeslåVedtaksperiode.kt

```diff
@@ -0,0 +1,67 @@
+package no.nav.tilleggsstonader.sak.vedtak
+
+import no.nav.tilleggsstonader.kontrakter.felles.Datoperiode
+import no.nav.tilleggsstonader.kontrakter.felles.Periode
+import no.nav.tilleggsstonader.kontrakter.felles.mergeSammenhengende
+import no.nav.tilleggsstonader.kontrakter.felles.overlapperEllerPåfølgesAv
+import no.nav.tilleggsstonader.sak.infrastruktur.exception.brukerfeil
+import no.nav.tilleggsstonader.sak.infrastruktur.exception.brukerfeilHvis
+import no.nav.tilleggsstonader.sak.vedtak.domain.Vedtaksperiode
+import no.nav.tilleggsstonader.sak.vilkår.stønadsperiode.ForeslåStønadsperiode.finnStønadsperioder
+import no.nav.tilleggsstonader.sak.vilkår.stønadsperiode.dto.StønadsperiodeDto
+import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.domain.Vilkår
+import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.domain.Vilkårsresultat
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.domain.Vilkårperioder
+import java.time.LocalDate
+
+object ForeslåVedtaksperiode {
+    fun finnVedtaksperiode(
+        vilkårperioder: Vilkårperioder,
+        vilkår: List<Vilkår>,
+    ): List<Vedtaksperiode> {
+        val stønadsperiode = finnStønadsperioder(vilkårperioder).single()
+        val oppfylteVilkår = vilkår.finnOppfylte()
+
+        brukerfeilHvis(oppfylteVilkår.isEmpty()) {
+            "Det finnes ingen vilkår som kan brukes i vedtaksperioden"
```

Comment:
Har hatt noen caser tidligere der saksbehandlerne ikke helt forstår hva som er feil ut fra en litt kryptisk feilmelding.  Tror kanskje dette også vil kunne være en sånn case. 

Hva med: 
```suggestion
            "Kunne ikke foreslå vedtaksperiode, ettersom det ikke er lagt inn noen utgiftsperioder der vilkårene er oppfylt."
```